### DISCUSSION TITLE
Copilot Test Generator Can't handle spaces in path

### DISCUSSION THREAD

**Discussion by MatteoAntolini** [community/community#52171] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I try to test a function in nodejs, copilot run this command 

`> node c:\Users\Matteo Antolini\Documents...`

Bacause I have a space in my user directory, it thorws this error

`Error: Cannot find module 'c:\Users\Matteo'`

**Comment by BREN0sx** [comment]
The issue here is that Copilot Test Generator cannot handle spaces in the path to the user's directory. To resolve this, the user can try enclosing the path in double quotes like this:

`node "c:\Users\Matteo Antolini\Documents..."`

This should allow Copilot to properly handle the space in the path and run the test without throwing an error.

**Reply by MatteoAntolini** [reply]
Hi, 
the command `node c:\Users\Matteo Antolini\Documents...` was run by copilot, not by me. The bug is that when it tries to run the test, it doesn't append double quotes on the command.

**Reply by BREN0sx** [reply]
If Github Copilot isn't adding the quotes automatically, you can try adding them manually in the code generated by Copilot. For example, you can try editing the command generated from `node c:\Users\Matteo Antolini\Documents...`  to `node "c:\Users\Matteo Antolini\Documents..."`.

Alternatively, you could try using the escape character \ before the space to let you know that the space is part of the path and that the quotes must be included, as I mentioned in a previous answer.

**Reply by MatteoAntolini** [reply]
Hi, I know I can do it manually, but doesn't this break the utility of copilot?
